subprojects/language-groovy/src/integTest/groovy/org/gradle/groovy/GroovyDocGenericsIntegrationTest.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.groovy
+
+import org.gradle.integtests.fixtures.TargetCoverage
+import org.gradle.testing.fixture.GroovydocCoverage
+import spock.lang.Issue
+
+@TargetCoverage({ GroovydocCoverage.GENERICS_ERROR_COVERAGE })
+class GroovyDocGenericsIntegrationTest {
+    @Issue(["GRADLE-3174", "GRADLE-3463"])
+    def "Error in Groovydoc generation is logged"() {
```

Comment:
This isn't part of a test now (since it doesn't extend `MultiVersionIntegrationSpec` or `Specification`).

I think this can remain a part of the other test. The important part wasn't that we were checking for failures due to the `<>`, we were trying to demonstrate that errors were redirected from the Ant groovydoc task to Gradle.  I looked around and couldn't find an easy way to get Groovydoc to fail with newer versions.

We might be OK just checking that we see output from `[ant:groovydoc]`. This seems to come out all the time:
> [ant:groovydoc] Resource org/codehaus/groovy/tools/groovy.ico not found so skipped

Either way, we should make sure this continues to work with newer versions of Groovy.